it/controllers/employment/PayeRefControllerISpec.scala

```diff
@@ -191,7 +186,6 @@ class PayeRefControllerISpec extends IntegrationTest with ViewHelpers with Emplo
           h1Check(expectedH1)
```

Comment:
You can actually delete line 173 to 202 and can remove second sentence from testName in line 145